Modules/CalcPerform.lua

```diff
@@ -789,21 +781,9 @@ function calcs.perform(env)
 			env.player.modDB:NewMod("GlobalWarcryCount", "BASE", numWarcries)
 			modDB:NewMod("AlreadyGlobalWarcryCooldown", "FLAG", true, "Config") -- Prevents effect from applying multiple times
 		end
-		if activeSkill.activeEffect.grantedEffect.name == "Summon Skeletons" then
-			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveSkeletonLimit")
-			output.ActiveSkeletonLimit = m_max(limit, output.ActiveSkeletonLimit or 0)
-		elseif activeSkill.activeEffect.grantedEffect.name == "Raise Zombie" then
-			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveZombieLimit")
-			output.ActiveZombieLimit = m_max(limit, output.ActiveZombieLimit or 0)
-		elseif activeSkill.activeEffect.grantedEffect.name == "Summon Raging Spirit" then
-			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveRagingSpiritLimit")
-			output.ActiveRagingSpiritLimit = m_max(limit, output.ActiveRagingSpiritLimit or 0)
-		elseif activeSkill.minionList and activeSkill.minionList[1] == "SummonedPhantasm" then
-			local limit = activeSkill.skillModList:Sum("BASE", activeSkill.skillCfg, "ActivePhantasmLimit")
-			output.ActivePhantasmLimit = m_max(limit, output.ActivePhantasmLimit or 0)
-		elseif activeSkill.activeEffect.grantedEffect.name == "Raise Spectre" then
-			local limit = env.player.mainSkill.skillModList:Sum("BASE", env.player.mainSkill.skillCfg, "ActiveSpectreLimit")
-			output.ActiveSpectreLimit = m_max(limit, output.ActiveSpectreLimit or 0)
+		if activeSkill.minion and activeSkill.minion.minionData and activeSkill.minion.minionData.limit then
+			local limit = m_floor(activeSkill.skillModList:Sum("BASE", nil, activeSkill.minion.minionData.limit))
```

Comment:
Why would you need to use `math.floor` here? Shouldn't `activeSkill.skillModList:Sum` always return a round value?

Reply:
Yeah, I think this is left over from copying a different line in while developing and I missed removing it.